File: assets/downloads/tutorials/extended-game-java/Drop.java
package com.badlogic.drop;

import com.badlogic.gdx.Game;
import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.graphics.g2d.BitmapFont;
import com.badlogic.gdx.graphics.g2d.SpriteBatch;
import com.badlogic.gdx.utils.viewport.FitViewport;


public class Drop extends Game {

	public SpriteBatch batch;
	public BitmapFont font;
	public FitViewport viewport;


	public void create() {
		batch = new SpriteBatch();
		font = new BitmapFont(); // use libGDX's default Arial font
		viewport = new FitViewport(8, 5);

		//font has 15pt, but we need to scale it our viewport by ratio of viewport height to screen height
		font.setUseIntegerPositions(false);
		font.getData().setScale(viewport.getWorldHeight() / Gdx.graphics.getHeight());

		this.setScreen(new MainMenuScreen(this));
	}

	public void render() {
		super.render(); // important!
	}

	public void dispose() {
		batch.dispose();
		font.dispose();
	}

}

File: assets/downloads/tutorials/extended-game-kotlin/Drop.kt
package drop

import com.badlogic.gdx.Game;
import com.badlogic.gdx.graphics.g2d.BitmapFont
import com.badlogic.gdx.graphics.g2d.SpriteBatch

class Drop : Game() {

  public lateinit var batch: SpriteBatch
  public lateinit var font: BitmapFont

  override fun create() {
    batch = SpriteBatch()
    // use LibGDX's default Arial font
    font = BitmapFont()
    this.setScreen(MainMenuScreen(this))
  }

  override fun render() {
    super.render()  // important!
  }

  override fun dispose() {
    // per @rohansuri's suggestion here:
    //    https://gist.github.com/sinistersnare/6367829#gistcomment-1661438
    this.getScreen().dispose()

    batch.dispose()
    font.dispose()
  }
}